.github/workflows/update-insecure-dependencies.yaml

```diff
@@ -0,0 +1,75 @@
+name: "Update insecure dependencies"
+
+on:
+  workflow_dispatch: { }
+  schedule:
+    - cron: 0 8 * * *
+jobs:
+  update-insecure-dependencies:
+    strategy:
+      matrix:
+        branch:
+          - "release-2.0"
+          - "release-1.8"
+          - "release-1.7"
+          - "release-1.6"
+          - "release-1.5"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Generate GitHub app token
+        id: github-app-token
+        uses: tibdex/github-app-token@021a2405c7f990db57f5eae5397423dcc554159c # v1.7.0
+        with:
+          app_id: ${{ secrets.APP_ID }}
+          private_key: ${{ secrets.APP_PRIVATE_KEY }}
+      - name: "Clone Kuma"
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ matrix.branch }}
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "~1.18.9"
+      - name: "Install tools"
+        run: |
+          sudo apt install -y jq git
+          GOBIN=/usr/local/bin/ go install github.com/google/osv-scanner/cmd/osv-scanner@v1
```

Comment:
git and jq should already be installed

Reply:
I was just testing it on docker ubuntu, I can remove it but if they're installed then this will just be a noop. With this we actually state what's needed and if they are removed for some reason this will make sure they are installed.

Reply:
I'd remove it, we don't do this kind a stuff in any other action and it's just increasing complexity for no reason